paper-api/src/main/java/io/papermc/paper/event/packet/PlayerBiomesLoadEvent.java

```diff
@@ -0,0 +1,78 @@
+package io.papermc.paper.event.packet;
+
+import org.bukkit.BiomesSnapshot;
+import org.bukkit.Chunk;
+import org.bukkit.entity.Player;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.world.ChunkEvent;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Nullable;
+import org.jspecify.annotations.NullMarked;
+
+/**
+ * Is called when a {@link Player} receives {@link org.bukkit.block.Biome}s
```

Comment:
```suggestion
 * Is called when a {@link Player} receives {@link org.bukkit.block.Biome Biomes}
```
Looks nicer in the javadocs

Reply:
The plural doesn't seem to work then. At least what I can see in your screen shoot.

Reply:
<img width="121" alt="Screenshot 2024-12-25 at 11 25 40 AM" src="https://github.com/user-attachments/assets/beede4fd-ad7b-42c5-b496-785252f51ca8" />
it puts the "s" as part of the word/link

Reply:
Thank you, changed it.